project/src/components/Layout/Layout.tsx

```diff
@@ -12,7 +12,7 @@ function Layout({ pageTitle = '', className = '', hasNav = true, children }: Lay
   return (
     <div className={`page ${className}`}>
       <Helmet>
-        <title>6 cities{`. ${pageTitle}`}</title>
+        <title>6 cities{` ${pageTitle}`}</title>
```

Comment:
В реакте можно без строки
```suggestion
        <title>6 cities {pageTitle}</title>
```